capsules/src/console.rs

```diff
@@ -63,6 +64,21 @@ pub struct App {
     read_len: usize,
 }
 
+impl GrantDefault for App {
+    fn grant_default(_process_id: ProcessId, cb_factory: &mut ProcessUpcallFactory) -> Self {
+        App {
+            write_callback: cb_factory.build_upcall(1).unwrap(),
```

Comment:
Shouldn't this start at 0?

Reply:
These values correctly represent the `subdriver_num` associated with each of those callbacks today. This PR does not require that subscribe numbers start at 0, only that they be declared in increasing order in this initialization function.